pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -35,6 +35,12 @@ section.bg-secondary { background-color: $brand_secondary_light; }
 .no-padding-bottom { padding-bottom: 0 !important;}
 
 // FontAwesome icon defaults
+// Sets fixed height on icon container to maintain consistent
+// spacing when icons have varying widths and heights
+i {
+  line-height: 1.25em !important;
```

Comment:
I would rewrite it to` i.fa-fw` and delete `!improtant`
This way it'll be more specific and in case we'll need to overwrite it - it'll be easier for us to do it in the future.

Reply:
Good call!

Reply:
Looks like this one is still in 'to do'